lib/cmdledit.g

```diff
@@ -700,28 +700,38 @@ BindKeysToGAPHandler("\022");
 BindGlobal("SaveCommandLineHistory", function(arg)
   local  fnam, append, hist, out;
 
-  if Length(arg) > 0 then
+  if Length(arg) = 1 then
+    if IsString(arg[1]) then
+      fnam := arg[1];
+      append := false;
+    elif arg[1] in [false, true] then
+      append := arg[1];
+    fi;
```

Comment:
you probably need an error case here, for when you don't understand arg1?

Reply:
Oh yeah, thanks!